examples/clusters/variables.tf

```diff
@@ -39,11 +39,11 @@ variable "subnet_id" {
 
 variable "api_stack_name" {
   type        = string
-  description = "The name of the cloudformation stack that deployed the ParallelCluster API."
+  description = "The name of the CloudFormation stack used to deploy the ParallelCluster API."
   default     = "ParallelCluster"
 }
 
 variable "api_version" {
   type        = string
-  description = "Version of the ParallelCluster API to deploy."
+  description = "The version of the ParallelCluster API."
```

Comment:
In `examples/api/variables.tf`, the description of api_version is: "The version of the ParallelCluster API to deploy."
I think you should keep the description consistent across the two files.

Reply:
The description is different because:
1. examples/api deploys a new PCAPI 
2. examples/clusters consumes an existing PCAPI